services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -1,98 +1,74 @@
 describe("Major Projects", () => {
-  beforeEach(() => {
-    cy.login();
+    beforeEach(() => {
+        cy.login();
 
-    cy.get('[data-cy="home-link-button-major-projects"]', { timeout: 5000 }).click({ force: true });
+        cy.get('[data-cy="home-link-button-major-projects"]', { timeout: 10000 }).click({ force: true });
```

Comment:
one thing- is this running 2 separate tests, the first which uploads the file and the second which downloads? If so, would it be faster to run it as one test that uploads and then downloads, taking out the login and navigation in the middle? I know that unit-test thinking promotes making tests smaller and independent, but cypress recommends combining pieces of the process into a longer one to enhance performance.

Reply:
Combined the 2 tests into one to enhance performance.